tests/test_edgeql_functions.py

```diff
@@ -3098,6 +3098,29 @@ async def test_edgeql_functions_to_str_07(self):
                 await self.con.query(
                     r'''SELECT to_str(<cal::local_time>'15:01:22', '');''',)
 
+    async def test_edgeql_functions_string_bytes_conversion(self):
+        encodings = ["utf-8", "koi8-u", "ISO_8859_5"]
+        string = "Паляниця"
+
+        for encoding in encodings:
```

Comment:
Would be great to see an "expect raises" test to make sure we don't leak SQL's NULLs as errors.

Reply:
(And that the errors we can propagate are actually mapped to something but an ISE)

Reply:
Done.